examples/pytorch/language-modeling/run_clm.py

```diff
@@ -101,6 +105,13 @@ class ModelArguments:
         },
     )
 
+    def __post_init__(self):
+        # convert "key1=val1,key2=val2" into a kwargs dict
+        if self.config_overrides is not None:
+            d = dict(x.split("=") for x in self.config_overrides.split(","))
+            # XXX: need more intelligent code to handle various types of numbers - currently just int
+            self.config_overrides = {k: int(v) for k, v in d.items()}
```

Comment:
The case should probably be done later on after instantiating a config and checking the type of all values to use the proper type?